Moli/Core/Screens/EditProfile/EditProfileView.swift

```diff
@@ -1,70 +1,211 @@
+            )
+            VStack(alignment: .leading) {
+                Text("About you")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+                LabelTextField(
+                    maxLimit: editProfileViewModel.STREAMER_NAME_MAX,
+                    minLimit: editProfileViewModel.STREAMER_NAME_MIN,
+                    label: "Name",
+                    text: $editProfileViewModel.editedStreamerName)
+                    .padding(.bottom, 8)
+                LabelTextField(
+                    maxLimit: editProfileViewModel.STREAMER_HANDLE_MAX,
+                    minLimit: editProfileViewModel.STREAMER_HANDLE_MIN,
+                    label: "Username",
+                    text: $editProfileViewModel.editedStreamerHandle)
+                    .padding(.bottom, 8)
+                VStack(alignment: .trailing) {
+                    LabelTextEditor(
+                        maxLimit: editProfileViewModel.BIOGRAPHY_MAX,
+                        label: "Bio",
+                        text: $editProfileViewModel.editedBiography,
+                        keyboardType: .twitter)
+                        .frame(minHeight: 80, alignment: .leading)
                 }
-                Section {
-                    AsyncButton(action: updateProfile) {
-                        Text("Update profile")
-                    }
+                .padding(.bottom, 16)
+
+                Text("Link")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+
+                LabelTextField(
+                    maxLimit: editProfileViewModel.WEBSITE_URL_MAX,
+                    placeHolder: "Link website",
+                    text: $editProfileViewModel.editedWebsiteUrl)
+                    .padding(.bottom, 8)
+
+                Text("Social")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+
+                if hasInstagram {
+                    SocialFieldView(label: "Unlink instagram", imageName: "Instagram")
+                        .onTapGesture(perform: toggleInstagramAlert)
+                        .alert(
+                            "Do you want to unlink your instagram? You can always link it back again",
+                            isPresented: $shouldShowRemoveInstagramAlert
+                        ) {
+                            Button("Yes", role: .destructive, action: onRemoveInstagram)
+                            Button("No", role: .cancel, action: toggleInstagramAlert)
+                        }
+                } else {
+                    SocialFieldView(label: "Link instagram", imageName: "Instagram")
+                        .onTapGesture(perform: onLinkInstagramPressed)
                 }
             }
         }
-        .onAppear(perform: setInitialValues)
+        .sheet(isPresented: $shouldShowInstagramWebView) {
+            WebView(viewModel: webViewModel)
+        }
+        .padding(.horizontal)
```

Comment:
:bomb: Det här beteende gör att man inte kan scrolla vid sidorna.. har lärt mig in the hard way xD det du kan göra är att du kan wrappa allt som finns i ScrollView:n i antingen en `Group` komponent, som är som en simpel Stack (V,H,Z stack) fanns utan någon spacing funktionalitet osv, så att man inte gör något med layout:n. Eller så kan du en VStack med spacing 0. Sen applicerar du .padding(.horizontal) som vanligt
```suggestion
        .padding(.horizontal)
```

Reply:
Snyggt fångat! Jag fixar